Gopkg.toml

```diff
@@ -0,0 +1,17 @@
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/bitrise-io/go-utils"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/bitrise-tools/go-steputils"
+
+[prune]
+  go-tests = true
+  unused-packages = true
+
+[[constraint]]
+  branch = "workspace_build_settings"
```

Comment:
shall we merge this and use master brach?

Reply:
yepp, @BirmacherAkos  needs to finish that PR and use `master` branch of `xcode-project` package